src/components/LeafletMap/LeafletMapContainer.js

```diff
@@ -68,7 +68,7 @@ const mapDispatchToProps = (dispatch) => {
             // fetch layer data for clicked point if needed
             let urls = layers.map(l => {
                 return GeoAPI.createUrl({
-                    layerName: l.layerName,
+                    layer: l.layer,
```

Comment:
If you need to pass the object layer, which I believe is the case, you should pass only the " l ". `l.layer` here is undefined, I think.